src/pyscanner.l

```diff
@@ -1205,51 +1205,49 @@ STARTDOCSYMS      "##"
 			current->initializer += " ";
                       }
    {B}                { // spaces
+			current->initializer += yytext;
 	              }
    {INTNUMBER}	      { // integer value
-			current->type = "int";
+			if (current-> type.isEmpty()) current->type = "int";
 			current->initializer += yytext;
-			BEGIN(VariableEnd);
 		      }
    {FLOATNUMBER}      { // floating point value
-			current->type = "float";
+			if (current->type.isEmpty()) current->type = "float";
 			current->initializer += yytext;
-			BEGIN(VariableEnd);
 		      }
    {BOOL}             { // boolean value
-			current->type = "bool";
+			if (current->type.isEmpty()) current->type = "bool";
 			current->initializer += yytext;
-			BEGIN(VariableEnd);
 		      }
    {STRINGPREFIX}?"'" { // string
-			current->type = "string";
+			if (current->type.isEmpty()) current->type = "string";
 			current->initializer += yytext;
 			g_copyString=&current->initializer;
-       			g_stringContext=VariableEnd;
+       			g_stringContext=VariableDec;
                         BEGIN( SingleQuoteString );
      		      }
    {STRINGPREFIX}?"\"" { // string
-			current->type = "string";
+			if (current->type.isEmpty()) current->type = "string";
 			current->initializer += yytext;
 			g_copyString=&current->initializer;
-       			g_stringContext=VariableEnd;
+       			g_stringContext=VariableDec;
                         BEGIN( DoubleQuoteString );
      		      }
    {TRIDOUBLEQUOTE}   { // start of a comment block
-			current->type = "string";
+			if (current->type.isEmpty()) current->type = "string";
```

Comment:
This makes a """-style comment appear as part of the initializer. I don't think that's right.